### DISCUSSION TITLE
Question: Is it possible to have a wildcard in a goal path?

### DISCUSSION THREAD

**Discussion by johanrosenson** [plausible/analytics#844] [Anything Else]
I am trying to set up a goal for a page that have a hash as part of the url, the hash part ofcourse changes everytime so i would need to have a wildcard for that portion.

I have tried a few different options to add a wildcard in the path when adding a goal, but so far no luck, `*` does not seem to work atleast.

Is this possible to do right now? Maybe i'm just guessing the syntax wrong :)

**Comment by Vigasaurus** [comment]
Yes! Assuming you're on the hosted version, that is. Self hosters on the stable track don't have this yet. 

https://plausible.io/docs/pageview-goals#pageview-goals-support-wildcards

**Reply by johanrosenson** [reply]
@Vigasaurus omg I'm sorry, i checked the documentation earlier when i was trying to set this up, but didn't see anything about it. is it a somewhat new functionality? It was about 2 months since i actually tried this, but just remembered to ask about it now.

Or was i equally blind two months ago? :) 

Thanks!

How can i mark this discussion as answered?

**Reply by Vigasaurus** [reply]
Haha yeah it's recent. 

I don't think you can mark this type of discussion solved, nbd.

**Reply by metmarkosaric** [reply]
thanks @Vigasaurus! and yeah marking as solved only works on self-hosted forum as it's a Q&A
